src/components/navigation/AppNavigationBoard.vue

```diff
@@ -34,39 +34,65 @@
 			style="opacity: 0.5" />
 
 		<template v-if="!deleted" slot="actions">
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="!showDueSettings"
+				icon="icon-more"
+				:close-after-click="true"
+				@click="actionDetails">
+				{{ t('deck', 'Board details') }}
+			</ActionButton>
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
 				icon="icon-rename"
 				:close-after-click="true"
 				@click="actionEdit">
 				{{ t('deck', 'Edit board') }}
 			</ActionButton>
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
 				icon="icon-clone"
 				:close-after-click="true"
 				@click="actionClone">
 				{{ t('deck', 'Clone board ') }}
 			</ActionButton>
-			<ActionButton v-if="canManage && board.archived"
+			<ActionButton v-if="canManage && board.archived && !showDueSettings"
 				icon="icon-archive"
 				:close-after-click="true"
 				@click="actionUnarchive">
 				{{ t('deck', 'Unarchive board ') }}
 			</ActionButton>
-			<ActionButton v-if="canManage && !board.archived"
+			<ActionButton v-if="canManage && !board.archived && !showDueSettings"
 				icon="icon-archive"
 				:close-after-click="true"
 				@click="actionArchive">
 				{{ t('deck', 'Archive board ') }}
 			</ActionButton>
-			<ActionButton v-if="canManage"
+
+			<ActionButton :icon="!showDueSettings ? 'icon-notifications-dark' : 'icon-view-previous' " @click="showDueSettings=!showDueSettings">
+				{{ t('deck', 'Due date reminders') }}
+			</ActionButton>
+			<ActionRadio v-if="showDueSettings"
+				name="notification"
+				:checked="board.settings['notify-due'] === 'all'"
+				@change="updateSetting('notify-due', 'all')">
+				All cards
+			</ActionRadio>
+			<ActionRadio v-if="showDueSettings"
+				name="notification"
+				:checked="board.settings['notify-due'] === 'assigned'"
+				@change="updateSetting('notify-due', 'assigned')">
+				Assigned cards
+			</ActionRadio>
+			<ActionRadio v-if="showDueSettings"
+				name="notification"
+				:checked="board.settings['notify-due'] === 'off'"
+				@change="updateSetting('notify-due', 'off')">
+				Off
```

Comment:
not translated :scream: 

This will be weird to translate, make `No notifications` is better?